tasks/2017/spread-the-word.md

```diff
@@ -0,0 +1,83 @@
+# GCI Tasks: Spread The Word
+
+## Background
+
+Upvotes and reviews make a big difference in the public perception of projects
+like Zulip.
+
+## Task description
+
+In this task, we will ask you to rate and review Zulip on some technology sites.
+You're free to decide whether or not to do each individual task, but we do ask
+that you do at least five of the social media marketing tasks in order to get
```

Comment:
If I counted correctly, there's only five tasks below. And I don't think many people can do both iOS and Android, so not sure if this `at least five` will work. Or maybe tasks are something else you had in mind?

Reply:
Ah, no. That is left over from when there used to be more tasks. Thanks for doing this read-through. Things changed enough that there was a lot to catch.